src/test-e2e/marketplace/marketplace.po.ts

```diff
@@ -0,0 +1,26 @@
+import { protractor, ElementArrayFinder, ElementFinder } from 'protractor';
+import { E2EHelpers } from '../helpers/e2e-helpers';
+import { browser, promise } from 'protractor';
+import { Page } from '../po/page.po';
+import { ListComponent } from '../po/list.po';
+
+export class MarketplacePage extends Page {
+  helpers = new E2EHelpers();
+
+  servicesList = new ListComponent();
+
+  navigateTo() {
+    return browser.get('/marketplace');
+  }
+
+  isMarketplacePage(strictCheck: boolean = true): promise.Promise<boolean> {
```

Comment:
Should be able to leverage existing page check on base class - not sure what your strictCheck does - seems to look for the URL just being the base URL + /.